packages/node_modules/@webex/plugin-meetings/test/unit/spec/meeting-info/meetinginfov2.js

```diff
@@ -98,6 +126,15 @@ describe('plugin-meetings', () => {
         });
       });
 
+      it('create adoc meeting when conversationUrl passed with enableAdhocMeetings toggle', async () => {
```

Comment:
should there be a test for the case when enableAdhocMeetings toggle is disabled?

Reply:
done